File: app/src/main/java/ssilvalucas/testeandroidv2/data/api/LoginService.java
package ssilvalucas.testeandroidv2.data.api;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;
import ssilvalucas.testeandroidv2.data.model.LoginResponse;

public interface LoginService {

    @FormUrlEncoded
    @POST("login")
    Call<LoginResponse> onLogin(@Field("user") String username, @Field("password") String password);
}

File: app/src/main/java/com/joaoneto/testeandroidv2/api/LoginService.kt
package com.joaoneto.testeandroidv2.api

import com.joaoneto.testeandroidv2.model.LoginResponseModel
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface LoginService {
    @FormUrlEncoded
    @POST("login")
    fun signIn(
        @Field("user") user: String,
        @Field("password") password: String
    ): Call<LoginResponseModel>
}